Taskfile.yml

```diff
@@ -143,6 +154,47 @@ tasks:
       vars:
         COMPONENT: "{{.TASK}}"
 
+  nodejs:
+    vars:
+      NODEJS_VERSION: "14.21.3"
+      TAR_FILE_NAME: "node-v14.21.3-linux-x64.tar.xz"
```

Comment:
@kirkrodrigues The node releases tar files do seem to follow the naming convention. Not sure if this is a regression by commit 393c1d7f - any reason we avoid this?
```yaml
      NODEJS_VERSION: "14.21.3"
      NODEJS_RELEASE: "v{{.NODEJS_VERSION}}-linux-x64"
      TAR_FILENAME: "node-{{.NODEJS_RELEASE}}.tar.xz"
      TAR_FILE_PATH: "{{.NODEJS_BUILD_DIR}}/{{.TAR_FILENAME}}"
```

Reply:
Although this seems to work, I couldn't find a guarantee that Go will respect the order of these defines meaning that if we define B in terms of A, this could break in the future. So I've been avoiding it.

Reply:
Gotcha. Thanks for the explanation.